app/src/main/java/com/isc/hermes/MainActivity.java

```diff
@@ -187,6 +185,17 @@ private void changeSearchView() {
         });
     }
 
+    /* Review 
```

Comment:
delete these comments or if it works for you later add a TODO

Reply:
Done.